skribenten-backend/src/main/kotlin/no/nav/pensjon/brev/skribenten/routes/Land.kt

```diff
@@ -1,14 +1,14 @@
 package no.nav.pensjon.brev.skribenten.routes
 
-import io.ktor.server.application.*
-import io.ktor.server.response.*
-import io.ktor.server.routing.*
-import java.util.*
+import io.ktor.server.response.respond
+import io.ktor.server.routing.Route
+import io.ktor.server.routing.get
+import java.util.Locale
 
-internal val norskBokmål: Locale = Locale("NB", "NO")
+internal val norskBokmål: Locale = Locale.of("NB", "NO")
 internal val landKoder: Set<String> = Locale.getISOCountries(Locale.IsoCountryCode.PART1_ALPHA2)
 internal val land = landKoder.map {
-    LandJson(it, Locale("", it).getDisplayCountry(norskBokmål))
+    LandJson(it, Locale.of("", it).getDisplayCountry(norskBokmål))
```

Comment:
Vet du hvorfor det brukes tom streng her?

Reply:
Skulle gjerne visst det. Innført i https://github.com/navikt/pensjonsbrev/pull/803 utan at det er heilt opplagt for meg korfor det er gjort sånn 🤔

Reply:
Lurer på om det er ein litt kronglete måte å seie `Locale.getDefault()` på, ser iallfall sånn ut ut frå koden.

Reply:
Ah, okey, det er iallfall ikkje intuitivt. Gjer ikkje noko med det no, er ikkje inne i konteksten her, men neste gong vi er inne her og faktisk gjer ting bør vi anten kommentere koden eller skrive om.